Zonk_2.0/GAME_LOGIC.c

```diff
@@ -7,57 +7,61 @@
 #include "GAME_CONST.h"
 
 /*������� �������� � �������� ��������� �������� ���������. ���� ������ ���������� - ���������� ������, � ��������� ������ - ����*/
-bool checkWin(int result)
+bool isWon(int result)
 {
 	return result > WINNING_RESULT - 1;
 }
 
 /*������ ���������� �������� ����� �� 1 �� 6*/
-int generatingRandomDice()
+int generateRandomDice()
 {
-	int number;
-	number = rand() % 5 + 1;
-
-	return number;
+	return rand() % 5 + 1;
 }
 
 
-/*������ �������� � �������� ��������� ���������� ���������������� ������ � ���������� ��������� �� ������ ��������� ������*/
-void fillRandomDices(int unusedDice, int(*fun)(), int* dices)
+/*������ �������� � �������� ��������� ��������� �� ������, ���������� ���������������� ������ � ������� ��������� ��������� �����*/
+void fillRandomDices(int* diceArray, int amountUnusedDice, int(*generateDiceValues)())
 {
 	int i;
 
-	if (dices != NULL)
+	if (diceArray != NULL)
 	{
-		for (i = 0; i < unusedDice; i++)
+		for (i = 0; i < amountUnusedDice; i++)
 		{
-			*(dices + i) = fun();
+			*(diceArray + i) = generateDiceValues();
 		}
 	}
 }
 
 
 /*������� �������� � �������� ��������� ��������� �� ������ ������ � ���������� ���������������� ������*/
-void printDices(int* dices, int unusedDice)
+void printDices(int* diceArray, int amountUnusedDice)
 {
 	int i;
 
-	for (i = 0; i < unusedDice; i++)
+	for (i = 0; i < amountUnusedDice; i++)
 	{
-		printf("|%d|  ", *(dices + i));
+		printf("|%d|  ", *(diceArray + i));
 	}
 }
 
 
-/*������� �������� � �������� ��������� ��������� �� ���������� �������������� ������ � ��������� �� ������ ��������� ������*/
-void playerMove(int* usedDice, int* selectedDice)
+/*������� �������� � �������� ��������� ��������� �� ������ ��������� ������*/
+void clearSelectedDice(int* selectedDice)
```

Comment:
What will happen if `selectedDice` size is less than `INITIAL_AMOUNT_DICE`? Pass array size as argument explicitly.
And the name confuses: does this function clear resources that it have taken or something else? Move it to `Utils.h`.